Server/Models/Item.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace Karma.Server.Models
+{
+    public class Item
```

Comment:
I believe we should have an interface named IGenericKarmaItem and then we could have different objects that implement said interface. Further down the line it could be FundraisingEvent, GiveawayItem, CharityAuction etc.

We can move this discussion to our messenger chat